run.py

```diff
@@ -411,6 +414,59 @@ def run(args):
                            docker_tag=docker_tag))
         service.start_launch(name=launch_name, start_time=timestamp(), description=launch_desc)
 
+    def fetch_test_details(var):
+        """Accepts the test and then provides the parameters of that test as a list
+        :param var: the test collected from the suite file
+        :return: Returns a dictionary of the various test params
+        """
+        details = dict()
+        details['docker-containers-list'] = []
+        details['name'] = var.get('name')
+        details['desc'] = var.get('desc')
+        details['file'] = var.get('module')
+        details['polarion-id'] = var.get('polarion-id')
+        polarion_default_url = "https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id="
+        details['polarion-id-link'] = "{}{}".format(polarion_default_url, details['polarion-id'])
+        details['rhbuild'] = rhbuild
+        details['ceph-version'] = ceph_version
+        details['ceph-ansible-version'] = ceph_ansible_version
+        details['compose-id'] = compose_id
+        details['distro'] = distro
+        details['suite-name'] = suite_name
+        details['suite-file'] = suite_file
+        details['conf-file'] = glb_file
+        details['ceph-version-name'] = ceph_name
+        details['duration'] = '0s'
+        details['status'] = 'Not Executed'
+        return details
+
+    tests = suite.get('tests')
+    res = []
+    for test in tests:
+        test = test.get('test')
+        tmp = fetch_test_details(test)
+        res.append(tmp)
+    project_dir = os.path.dirname(os.path.abspath(__file__))
+    template_dir = os.path.join(project_dir, 'templates')
+    env = Environment(
```

Comment:
would suggest to make use of utility/util.py for pre-build/post-build action definitions.

Reply:
so we could avoid calling other packages like jinja2 here again

Reply:
added method in utils.py and cleared the calls from run.py